v1/feed.js

```diff
@@ -193,7 +197,7 @@ Feed.prototype.aggregated = function(hyper, req) {
             }
 
             if (feed.random) {
-                feed.random.items = assignAllSummaries(feed.random.items);
+                feed.random = assignAllSummaries(feed.random.items);
```

Comment:
I asked the same question, but it seems that the Android app is not using this property yet, and IOS will be the first to start using this soon. Would be good to have confirmation from @berndsi, though.